File: C Modules/Assignments/Assignment no 7/Question-1/Type_3/Assignment no 2/Question-1.c
#include<stdio.h>
int calculate()
{
    int a,b;
    char op;
    int *ptr1=&a;
    int *ptr2=&b;

    printf("enter first number:");
    scanf("%d",ptr1);
    printf("enter second number:\n");
    scanf(" %d",ptr2);
    printf("enter the operators:\n");
    scanf(" %c", &op);
      if (op=='+')
    {
        return *ptr1+*ptr2;
    }
    
    else if (op=='-')
    {
       return *ptr1-*ptr2;
    }

    else if (op=='*')
    {
        return *ptr1**ptr2;
    }

    else if (op=='/'){
    if (b!=0)
    {
        return *ptr1 / *ptr2;
    }
    else{
        printf("%d is not divisible by 0\n");
    }

    }
    else if (op=='%')
    {
        if (*ptr2!=0)
        {
          return *ptr1%*ptr2;
        }
        else{
            printf("%d is not modulo by 0\n"); 
        }
    }

    else{
        printf("Invalid operator\n");
        return 0;
    }
}
int main()
{
   int result=calculate();
   printf("Result=%d",result);
    return 0;
}

File: Java Module/Assignments in c to java/Assignment-02/Question_1.java
package 
public class Question_1 {

    public static void main(String args[])
    {
        
    int a=10,b=10;
    char op='*';

    // printf("enter first number:");
    // scanf("%d",&a);
    // printf("enter second number:\n");
    // scanf(" %d",&b);
    // printf("enter the operators:\n");
    // scanf(" %c", &op);
    if (op=='+')
    {
        System.out.printf("result=%d\n",a+b);
        /* code */
    }
    
    else if (op=='-')
    {
        System.out.printf("result=%d\n",a-b);
        /* code */
    }

    else if (op=='*')
    {
        System.out.printf("result=%d\n",a*b);
        /* code */
    }

    else if (op=='/'){
    if (b!=0)
    {
        System.out.printf("result=%d\n",a/b);
        /* code */
    }
    else{
        System.out.printf("%d is not divisible by 0\n");
    }

    }
    else if (op=='%')
    {
        if (b!=0)
        {
            System.out.printf("result=%d\n",a%b);
            /* code */
        }
        else{
            System.out.printf("%d is not modulo by 0\n"); 
        }
        /* code */
    }


    }
}